sdk/core/azure-core-amqp/README.md

```diff
@@ -24,6 +25,16 @@ target_link_libraries(<your project name> PRIVATE Azure::azure-core-amqp)
 
```

Comment:
Look at the things above - we don't say that using `target_include_directories(<your project name> PRIVATE ${azure-core-amqp_SOURCE_DIR})` is even necessary. Readme.md should have the correct instructions.